### ISSUE TITLE
Allow catalog owners to add editors

### ISSUE THREAD

**Issue by ebner** [MetaSolutionsAB/dcat-manager#42]
Requires a basic admin-GUI for catalogs; to search for other existing users in the system and give access rights to them.

**Comment by thomaskvist** [comment]
Agree. We also would like the possiblity to have two administrators having maximun rights on the same catalog. We want to avoid a dependence to only one person.
